Snowdog_SolrNavigation/templates/filters/list.phtml

```diff
@@ -4,10 +4,11 @@
         <a
             href="<?= $block->getCleanUrl($valueId) ?>"
             class="active-filters__remove"
-            aria-label="<?= __('Disable filter') ?>"
-            title="<?= __('Disable filter') ?>"
+            aria-label="<?= __('Disable filter') . ': ' . $valueLabel ?>"
```

Comment:
I'I'm affriad that using aria-label in this element, the `close` icon label won't be read. It's not a bug indeed, because the text doesn't repeat but you can check if there is the reason to use aria-label

Reply:
this aria-label is important, I set `close` to `role="presentation"`

Reply:
👍